index.js

```diff
@@ -1 +1,72 @@
 // implement your API here
+
+const express = require('express');
+const db = require('./data/db');
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/api/users', (req, res) => {
+    const user = req.body;
+
+    db.insert(user)
+        .then(users => {
+            if(!user.name || !user.bio) {
+                res.status(400).json({ success: false, errorMessage: 'PLease provide name and bio for the user.'});
+            } else {
+                res.status(201).json({ success: true, users });
+            }
+        })
+        .catch(err => {
+            res.status(500).json({ success: false, error: 'There was an error while saving the user to the database. ', err});
```

Comment:
you'll get an empty err object `err: {}` if you get an error. What I would do is do `err: err.message` so that it at least gives you a helpful error message in addition to the message you are giving.